docs/installing/README.md

```diff
@@ -25,28 +25,42 @@ nix-env -iA nixos.starship
 
 #### Declarative, single user, via [home-manager](home-manager)
 
-Add `pkgs.starship` to your `home.packages` in your `home.nix` file, then run
+Enable the `programs.starship` module in your `home.nix` file, and add your settings
+
+```nix
+{
+  programs.starship = {
+    enable = true;
+    enableZshIntegration = true;
+    settings = {
```

Comment:
I'm not familiar with how Nix works here. Do these settings eventually become part of the starship config file? If so, are they necessary to set here, or is this just a sane default set?

Reply:
They're just the example config [that HM gives](https://rycee.gitlab.io/home-manager/options.html#opt-programs.starship.settings). The HM module [generates the Starship config file](https://github.com/nix-community/home-manager/blob/master/modules/programs/starship.nix#L96) from the `settings` entry.

Reply:
Here's an adapted version of [the example on the Starship docs](https://starship.rs/config/) with every option commented.

```nix
{
  settings = {
    # add_newline = false;

    # character = {
    #   success_symbol = "[➜](bold green)";
    #   error_symbol = "➜(bold red)";
    # };

    # package.disabled = true;
  };
}
```

Would this be better for the HM example?

Reply:
I believe 
```
#   error_symbol = "➜(bold red)";
```

needs to be

```
#   error_symbol = "[➜](bold red)";
```

but otherwise yes, I think that would be a good config. It might also be nice to add a comment in that block explaining that this is for the config/settings and can be set to change the generated starship.toml? Not sure how to phrase that in a concise way.

Reply:
Indeed, that's a typo. And using the description in the HM manual for the settings entry.

```nix
{
  # Configuration written to ~/.config/starship.toml
  settings = {
    # add_newline = false;

    # character = {
    #   success_symbol = "[➜](bold green)";
    #   error_symbol = "[➜](bold red)";
    # };

    # package.disabled = true;
  };
}
```

Reply:
That looks good!